sdk/src/README.md

```diff
@@ -0,0 +1,594 @@
+# C2PA SDK architecture
+
+This document describes the outermost architectural layer of the c2pa SDK. The SDK implements the C2PA (Coalition for Content Provenance and Authenticity) specification, providing functionality to create, embed, read, and validate C2PA manifests in digital assets.
+
+## Architectural overview
+
+The SDK follows a layered architecture that separates concerns between user-facing APIs, data models, storage, asset I/O, cryptography, and validation.
```

Comment:
I would call `user-facing APIs` -> `public APIs`. It's what they are,e ventually.